### ISSUE TITLE
Can't update profile for tvOS

### ISSUE THREAD

**Issue by sanderlooijenga** [pavjacko/renative#308]
**Describe the bug**
It's not possible to update a profile for tvOS, even though specifiying `-p tvos`, it will only update it for iOS.

**To Reproduce**
1. Run command `npx rnv crypto updateProfile -p tvos -s alpha`
2. In log you will see;

```
+-------------------------------------+-----------------------------------+
|                        Summary for sigh 2.133.0                         |
+-------------------------------------+-----------------------------------+
| app_identifier                      | <app_identifier> |
| team_id                             | <TEAM_ID>                        |
| force                               | true                              |
| adhoc                               | false                             |
| developer_id                        | false                             |
| development                         | false                             |
| skip_install                        | false                             |
| ignore_profiles_with_different_name | false                             |
| skip_fetch_profiles                 | false                             |
| skip_certificate_verification       | false                             |
| platform                            | ios                               |
| readonly                            | false                             |
+-------------------------------------+-----------------------------------+
```

**Expected behavior**
In the aforementioned table output `platform` should be set to `tvos` (not `ios`) ?

**Desktop (please complete the following information):**
 - OS: osx
 - Node Version: v11.10.0
 - RNV Version:  rnv v0.27.5-alpha.4

**Comment by mihaiblaga89** [comment]
@sanderlooijenga I don't think we can do anything about it for now, `sigh` only supports ios and mac. I don't see why it wouldn't support webos, maybe we can make a PR on their side.
<img width="574" alt="Screenshot 2019-10-08 at 18 46 12" src="https://user-images.githubusercontent.com/4787898/66415838-ce14a280-e9fc-11e9-9a21-ede5f0ce5f8f.png">

**Comment by mihaiblaga89** [comment]
well, apparently if you specify tvos as platform it works :D closed by #317

**Comment by sanderlooijenga** [comment]
![Alt Text](https://media.giphy.com/media/P5vjklHg5MqqY/source.gif)
